acp/ideas_module.php

```diff
@@ -93,11 +93,12 @@ public function main($id, $mode)
 		$display_vars = array(
 			'ideas_forum_id'	=> array('lang' => 'ACP_IDEAS_FORUM_ID',	'validate' => 'string',	'type' => 'custom', 'method' => 'select_ideas_forum', 'explain' => true),
 			'ideas_poster_id'	=> array('lang' => 'ACP_IDEAS_POSTER_ID',	'validate' => 'string',	'type' => 'custom', 'method' => 'select_ideas_topics_poster', 'explain' => true),
+			'ideas_forum_setup'	=> array('lang' => 'ACP_IDEAS_FORUM_SETUP',	'validate' => 'bool',	'type' => 'custom', 'method' => 'set_ideas_forum_permissions', 'explain' => true),
 		);
 
 		$this->new_config = $this->config;
 		$cfg_array = ($this->request->is_set('config')) ? $this->request->variable('config', array('' => ''), true) : $this->new_config;
-		$submit = $this->request->is_set('submit');
+		$submit = $this->request->is_set('submit') || $this->request->is_set_post('ideas_forum_setup');
```

Comment:
ideas_forum_setup should be it's own isolated action, and not submit, otherwise we re-validate and resubmit the other fields too..